kivy_garden/graph/__init__.py

```diff
@@ -1527,6 +1527,37 @@ def plot_mesh(self, *args):
             vert[k * 8 + 5] = px_ymax
         mesh.vertices = vert
 
+class ScatterPlot(Plot):
+    """
+    ScatterPlot draws using a standard Point object.
+    The pointsize can be controlled by passing `pointsize` when initialising
```

Comment:
Instead of only listening to the value that was originally passed in, it should be responsive to changes e.g. `my_plot.point_size = 3`, should update the plot. See how the other plots do it (e.g. https://github.com/kivy-garden/graph/blob/master/kivy_garden/graph/__init__.py#L1234).

Reply:
Now it responds to changes in `point_size`